src/annotator/annotation-sync.js

```diff
@@ -108,7 +110,7 @@ export class AnnotationSync {
     if (annotation.$tag) {
       return annotation;
     }
-    tag = tag || window.btoa(Math.random().toString());
+    tag = tag || generateHexString(8);
```

Comment:
The only question I have is about this line.

This creates around 24 character long string of random alpha-numerical characters. This was different from `src/sidebar/services/annotations.js` that use `generateHexString(8)`.

Is there any value on having a different format for the `$tag` here than the one generated in the sidebar?